### ISSUE TITLE
enhance the questioner experience 

### ISSUE THREAD

**Issue by SanliFaez** [Klimaat-Helpdesk/website#5]
comments from a test user: 

Ik vind ‘bedankt, we houden je op de hoogte’; nog wat mager. Je zou een automatische email moeten krijgen als je je mailadres hebt achtergelaten waarin ook staat hoe lang het max duurt voordat je antwoord krijgt. 

To do: Andere tekst voor auto-response
To do: Automatische mailing nadat je een vraag hebt ingediend

Ik vind een maand nogal lang overigens. 

To do: Misschien tussendoor vragenstellers een mail sturen met zoiets als: “We hebben een expert gevonden die jouw vraag gaat beantwoorden."

Dat doen alle helpdesks normaal gesproken ook, vaak met een call nummer waaraan je kan refereren bij correspondentie.

**Comment by aquilesC** [comment]
I agree, we could send automatic e-mails after receiving a question from a user. 

Doubts: Who writes the text and where is it kept? 

> Ik vind een maand nogal lang overigens.

This may be because of an unclear value proposition on the website. I think this may arise from a disconnect between our group (i.e. people used to scientific publishing timelines) and the public we are targetting (used to getting answers in less than 24 hours). A simple solution is not to mention any timeline, and just keep them in the loop of the progress periodically (or not, in any case the only relevant moment is when the question is answered). 

For the second part: 

> Misschien tussendoor vragenstellers een mail sturen met zoiets als: “We hebben een expert gevonden die jouw vraag gaat beantwoorden."

This could be handled "manually" by the editor responsible for tracking the progress of an answer (several canned responses can be sent at different stages). Without knowing the volume of questions/answers and timeframes, it is not worth automatizing preemptively.  

> Dat doen alle helpdesks normaal gesproken ook, vaak met een call nummer waaraan je kan refereren bij correspondentie.

This is true for high volume call centers that need to pick up the issue through different entry points. We should be careful not to overcomplicate things too much.

**Comment by SanliFaez** [comment]
agree with all points Aquiles, automating at this stage is not so productive.

emailing about the procedure to users have to be done manually, because they might involve discussions on the content of the question. we can keep some template answers on wiki.

I have change "we need 4 weeks" to "on average 2 weeks" to avoid the shocking effect.

**Comment by JoselineHouwman** [comment]
Maybe change the "bedankt, we houden je op de hoogte" to "Bedankt voor het stellen van je vraag. We nemen je vraag in behandeling  en proberen zo snel mogelijk een expert te vinden die je vraag kan beantwoorden."
This is a bit friendlier.

**Comment by JoselineHouwman** [comment]
Also, when asking a question, you don't necessarily need to select a category. If you don't, it says "een nieuwe categorie" on the next page.
Maybe we could add a sentence to the first ask-a-question-page: "Als je niet weet bij welke categorie je vraag hoort, selecteer dan geen van de categoriën. Wij zullen dan naderhand de vraag in een categorie indelen."

**Comment by SanliFaez** [comment]
another suggestion which might also help many other visitors other than the person asking the question: to report the process of handling the incoming questions on twitter. 
this way, we can add on the thanks page: "you can follow the handling process of your question and those of others on our twitter feed"

**Comment by aquilesC** [comment]
These are great ideas! I wonder what is the best way to keep track of them so we can have a bit more atomic task-list.  

Something that I considered a while ago (the very early version of the website ~Dec2019) was to include the editorial process on the website and away from Gitlab. This opens the door to infinite degrees of freedom, such as automatic remainders to editors, better overview of the progress, automatic Twitter posts about progress, we could even accept questions through Twitter and display them on a dashboard, etc.
